packages/binary-assets/README.md

```diff
@@ -13,6 +13,10 @@ Returns the full URL of the asset at `path` managed by the
 assetUrl("some/blob.png") // -> "https://<origin>/some/blob.png"
 ```
 
+```scss
+asset-url("some/blob.png") // -> "https://<origin>/some/blob.png"
+```
+
```

Comment:
Now that there are 2 examples, I think it'd be helpful for people who stumble on this for the first time to see labels above each example saying "JavaScript" and "SCSS". Otherwise the kebab case is the only thing to tell them it's SCSS (on GitHub's Markdown preview).